packages/wrangler/src/__tests__/versions/secrets/utils.ts

```diff
@@ -82,8 +82,12 @@ export function mockGetVersion(versionInfo?: VersionDetails) {
 function mockGetVersionContent() {
 	msw.use(
 		http.get(
-			`*/accounts/:accountId/workers/scripts/:scriptName/content/v2?version=ce15c78b-cc43-4f60-b5a9-15ce4f298c2a`,
```

Comment:
This was causing MockServiceWorkers (MSW) to write warnings to the console about a redundant query in the URL matcher.